### ISSUE TITLE
Repair as fugitive

### ISSUE THREAD

**Issue by coolman5785** [my-name-is-samael/BeamJoy#62]
Is there a way to make the repair station work as the fugitive?

**Comment by my-name-is-samael** [comment]
Good point. I feel like it would be cheating for the fugitive to have a way to repair, since their skill must be to drive fast and safer than hunters. I need to think about this.
That could ba a launch setting option since some maps have a lot of garages and some others have way less...

**Comment by coolman5785** [comment]
Having it as a setting would be the best option. My friends does crash regularly as the fugitive.

**Comment by BadmanTru** [comment]
I would suggest just adding gas stations as repair areas. I think that would be best.

**Comment by my-name-is-samael** [comment]
applied in the next release, a fugitive will have the ability to respawn if hunters are not nearby (would be cheating otherwise)
